typegraph/python/typegraph/runtimes/grpc.py

```diff
@@ -0,0 +1,33 @@
+# Copyright Metatype OÜ, licensed under the Mozilla Public License Version 2.0.
+# SPDX-License-Identifier: MPL-2.0
+
+from typegraph import t
+from typegraph.runtimes.base import Runtime
+from typegraph.gen.exports.runtimes import (
+    GrpcData,
+    GrpcRuntimeData,
+)
+from typegraph.gen.types import Err
+from typegraph.wit import runtimes, store
+
+
+class GrpcRuntime(Runtime):
+    def __init__(self, proto_file: str, endpoint: str):
+        file = open(proto_file, "r")
+        proto_file_content = file.read()
+        file.close()
```

Comment:
I suggest you to rely on [fs.rs](https://github.com/metatypedev/metatype/blob/main/src/typegraph/core/src/utils/fs.rs#L123), this way you don't have to fight over workdir normalization accross the SDKs (something like [this](https://github.com/metatypedev/metatype/blob/7a221f62cef4dce2f9f2f961f0a5a855134ac22c/src/typegraph/core/src/utils/postprocess/deno_rt.rs#L23).